tests/config_override_test.py

```diff
@@ -257,3 +258,74 @@ def teardown_class(cls):
         print("TEARDOWN")
         os.environ["UTILITIES_UNIT_TESTING"] = "0"
         return
+
+
+class TestConfigOverrideMultiasic(object):
+    @classmethod
+    def setup_class(cls):
+        print("SETUP")
+        os.environ["UTILITIES_UNIT_TESTING"] = "1"
+        os.environ["UTILITIES_UNIT_TESTING_TOPOLOGY"] = "multi_asic"
+        # change to multi asic config
+        from .mock_tables import dbconnector
+        from .mock_tables import mock_multi_asic
+        importlib.reload(mock_multi_asic)
+        dbconnector.load_namespace_config()
+        return
+
+    def test_macsec_override(self):
+        def read_json_file_side_effect(filename):
+            return macsec_profile
+        db = Db()
+        cfgdb_clients = db.cfgdb_clients
+        macsec_profile = {}
```

Comment:
This  will work .. 

but I suggest to create a golden_config_db.json file in the tests directory  in the format we need ( where we have host + various namespaces -- and th emacsec profile data in each of asic0,asic1 etc namespaces )

Reply:
fixed.